css-brochure-form/styles.css

```diff
@@ -49,11 +41,14 @@ button {
 
 label {
   font-family: 'AtlasGrotesk', sans-serif;
-  padding-left: 13px;
-  line-height: 25px;
   font-size: 13px;
 }
 
+h3 {
+  padding-left: 10px;
+  margin-bottom: 10px;
+}
```

Comment:
Instead of using an `h3` and changing their behavior from the default, why not just make the label bold as I had mentioned in my feedback?